po/de.po

```diff
@@ -2405,6 +2408,12 @@ msgid ""
 "<b>N.B.</b> This is incompatible with previous versions of the report and "
 "will require regenerating saved configurations."
 msgstr ""
+"Die Unterstützung für die britische Mehrwertsteuer und die australische GST "
+"wurde dem Bericht \"Einkommen-GST\" hinzugefügt. Die Optionen des Berichts "
+"werden von Quellkonten zu Quellverkaufs- und Einkaufskonten geändert, um "
+"eine ordnungsgemäße Meldung von Kapitalkäufen zu ermöglichen. <b> N.B.</b> "
+"Dies ist mit früheren Versionen des Berichts nicht kompatibel und erfordert "
+"eine Neugenerierung der gespeicherten Konfiguration."
```

Comment:
How well known is the abbreviation N.B.?
According to the relevant reference books, it stands for "nota bene (lt.)" which means "note well" or "take notice".